foreign_cc/private/shell_toolchain/toolchains/impl/default_commands.bzl

```diff
@@ -61,7 +61,7 @@ fi
     )
 
 def copy_dir_contents_to_dir(source, target):
-    return """cp -L -p -r --no-target-directory "{}" "{}" """.format(source, target)
+    return """cp -L -r --no-target-directory "{source}" "{target}" && find {target} -type f -exec touch -r "{source}" "{{}}" \\;""".format(source=source, target=target)
```

Comment:
Can you move the parameters to the next so the line isn't so long?

```python
"".format(
    source = source,
    target = target,
)
```
🙏

Reply:
Done! Thanks :)